css/style.scss

```
@@ -101,14 +109,70 @@
 	display: inline-block;
 }
 
-#commentsTabView_header > p {
+#commentsTabView_header>p {
 	margin: 10px;
 	font-size: 22px;
 	font-weight: bold;
 	color: var(--color-text-light);
 	display: inline-block;
 }
 
+.collapsible {
+	cursor: pointer;
+	padding: 18px;
+	width: 100%;
+	border: none;
+	text-align: left;
+	outline: none;
+	font-size: 15px;
+	border-radius: 5px;
+}
+
+.collapsible * {
+	cursor: pointer;
+}
+
+.active,
+.collapsible:hover,
+.collapsible:hover * {
+	background-color: #aaa;
```

this does not work great with dark mode,
can you use this instead:

```suggestion
	background-color: var(--color-background-hover);
```
or
```suggestion
	background-color: var(--color-background-dark);
```